public/css/list/schedule-list.less

```diff
@@ -0,0 +1,76 @@
+// Header
+.schedules-header {
+  margin-left: 12em;
+  width: ~"calc(100% - 12em)";
+
+  .days-header {
+    display: grid;
+    grid-template-columns: repeat(7, minmax(2em, 1fr));
+
+    .column-title {
+      border-right: 1px solid @gray-lighter;
+    }
+
+    .column-title:first-of-type {
+      border-left: 2px solid @gray-lighter
```

Comment:
Apply this to `.days-header` instead, otherwise the alignment of the other borders is always off.